util/log/src/main/scala/sbt/ConsoleLogger.scala

```diff
@@ -35,6 +35,19 @@ object ConsoleLogger {
   def isEscapeTerminator(c: Char): Boolean =
     c >= '@' && c <= '~'
 
+  /**
+   * Test if the character AFTER an ESC is the ANSI CSI, which means more than two characters until terminatioin.
+   */
+  def isCSI(c: Char): Boolean = c == '['
+
+  /**
+   * Tests whether or not a character needs to immediately terminate the ANSI sequence.
+   */
+  def isSimpleEscapeTerminator(c: Char): Boolean = {
+    val n = c.toInt
+    (n >= 64) && (n <= 95)
```

Comment:
Also would it make sense to write `'@'` instead of `64` to be consistent with the `isEscapeTerminator` above?